sbin/get-redisjson

```diff
@@ -82,6 +82,13 @@ else
 		else
 			nick="rhel7"
 		fi
+	elif [[ $dist == mariner ]]; then
+		OSS=1
```

Comment:
What `OSS=1` means? Why do we need it?

Reply:
We need it because RedisJSON is being uploaded to `rejson-oss/snapshots/` directory